src/mixins/common-mixin.ts

```diff
@@ -40,13 +40,14 @@ export class CommonMixin extends Vue {
       case EntityTypes.SP: return 'BC Sole Proprietorship'
       case EntityTypes.UL: return 'BC Unlimited Liability Company'
 
-      // Continuation In Entity Types:
-      case EntityTypes.C: return 'Continuation In (BC Limited Company)'
-      case EntityTypes.CBEN: return 'Continuation In (Benefit Company)'
-      case EntityTypes.CCC: return 'Continuation In (BC Community Contribution Company)'
-      case EntityTypes.CUL: return 'Continuation In (BC Unlimited Liability Company)'
+      // Continuation In entity types:
+      case EntityTypes.C: return 'BC Limited Company (Continuation In)'
+      case EntityTypes.CBEN: return 'Benefit Company (Continuation In)'
+      case EntityTypes.CCC: return 'BC Community Contribution Company (Continuation In)'
+      case EntityTypes.CS: return 'BC Social Enterprise (Continuation In)'
+      case EntityTypes.CUL: return 'BC Unlimited Liability Company (Continuation In)'
```

Comment:
I added CS for completeness (also in 2 other methods below).

I changed the text because the name is more important than the type, but this really doesn't matter because these strings are not used in Namerequest UI.

Example (where name and type are displayed separately):

![image](https://github.com/bcgov/namerequest/assets/10002889/66bdd1cc-c504-42a9-a664-3cbcf758f4ef)